### PR TITLE
feat: add tests according to the task

### PR DESCRIPTION
**Pull request by AlenaKharytonchyk**
Done according to the task:

 Counter

Test that component renders initial value provided in props
Test that a click event on "decrement" button decrements the displayed value
Test that a click event on "increment" button increments the displayed value

SearchForm

Test that component renders an input with the value equal to initial value passed in props
Test that after typing to the input and a "click" event on the Submit button, the "onChange" prop is called with proper value
Test that after typing to the input and pressing Enter key, the "onChange" prop is called with proper value

GenreSelect

Test that component renders all genres passed in props
Test that component highlights a selected genre passed in props
Test that after a click event on a genre button component calls "onChange" callback and passes correct genre in arguments

### CODE REVIEW

File: src/components/counter/counter.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Counter } from "../../components";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+describe("Counter", () => {
+    it("renders initial value provided in props", () => {
+        render(<Counter initialValue={0} />);
+        const initialValue = screen.getByText("0");
```
Comment (AliaksandrZahorski) [inline src/components/counter/counter.test.js]: queryByText

File: src/components/genreSelect/genreSelect.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { GenreSelect } from "../../components";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+describe("GenreSelect", () => {
+    it("renders all genres passed in props", () => {
+        const genreList = ["all", "comedy", "drama", "detective"];
```
Comment (AliaksandrZahorski) [inline src/components/genreSelect/genreSelect.test.js]: why we need 3 copy of this data?

File: src/components/input/input.test.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { InputField } from "../../components";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+describe('SearchForm', () => {
+    it("renders input with initial value provided in props", () => {
+        const initialValue = "it is test value";
```
Comment (AliaksandrZahorski) [inline src/components/input/input.test.js]: DRY

File: src/components/genreSelect/genreSelect.test.js (hunk 2 above)
Comment (AlenaKharytonchyk) [reply src/components/genreSelect/genreSelect.test.js]: Fixed

File: src/components/input/input.test.js (hunk 3 above)
Comment (AlenaKharytonchyk) [reply src/components/input/input.test.js]: Fixed

File: src/components/counter/counter.test.js (hunk 1 above)
Comment (AlenaKharytonchyk) [reply src/components/counter/counter.test.js]: Done, than you

**Comment by AliaksandrZahorski** [comment]
Approved

**Pull request merged**